XPlot.Plotly.NET.Interactive/KernelExtension.fs

```diff
@@ -0,0 +1,50 @@
+﻿namespace XPlot.Plotly.NET.Interactive
+
+open System
+open System.Text
+open System.IO
+open System.Threading.Tasks
+
+open Microsoft.AspNetCore.Html
+open Microsoft.DotNet.Interactive
+open Microsoft.DotNet.Interactive.Formatting
+open Microsoft.DotNet.Interactive.Http
+open XPlot.Plotly
+
+open Giraffe.ViewEngine
+
+type KernelExtension() =
+    let getScriptElementWithRequire (script: string) =
+        let newScript = StringBuilder()
+        newScript.AppendLine("""<script type="text/javascript">""") |> ignore
+        newScript.AppendLine("""
+var renderPlotly = function() {
+    var xplotRequire = require.config({context:'xplot-3.0.1',paths:{plotly:'https://cdn.plot.ly/plotly-1.49.2.min'}}) || require;
+    xplotRequire(['plotly'], function(Plotly) {" """) |> ignore
+        newScript.AppendLine(script) |> ignore
+        newScript.AppendLine(@"});
+};"
+        ) |> ignore
+        newScript.AppendLine(JavascriptUtilities.GetCodeForEnsureRequireJs(Uri("https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.6/require.min.js"), "renderPlotly")) |> ignore
+        newScript.ToString()
```

Comment:
Just for fun, this can now be an interpolated string since it's using the .NET 5 SDK